src/bindings.c

```diff
@@ -4,9 +4,17 @@
 #define _GNU_SOURCE
 #endif
 
+#include "config.h"
+
+#ifdef HAVE_FUSE3
+#ifndef FUSE_USE_VERSION
+#define FUSE_USE_VERSION 30
+#endif
+#else
 #ifndef FUSE_USE_VERSION
 #define FUSE_USE_VERSION 26
 #endif
+#endif
```

Comment:
I wonder if we shouldn't just encapsulate this in a common header `fuse_version.h` instead of duplicating this in every file.

Reply:
Similar to what you already did with `fuse_compat.h` below.